src/convertToObject.js

```diff
@@ -12,7 +13,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssNormalized = {};
+
+  sourceString.split(';').forEach((element) => {
+    const [property, value] = element.split(':');
+
+    if (property && value) {
+      const normalProperty = property.replace('\n', '').trim();
```

Comment:
I think it would be better to rename from `normalProperty`  to `formattedProperty`